src/main/java/bio/overture/rollcall/repository/IndexRepository.java

```diff
@@ -22,39 +22,44 @@
 import lombok.val;
 import org.elasticsearch.action.admin.indices.alias.IndicesAliasesRequest;
 import org.elasticsearch.action.admin.indices.alias.get.GetAliasesRequest;
-import org.elasticsearch.action.admin.indices.get.GetIndexRequest;
-import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.GetIndexRequest;
 import org.elasticsearch.cluster.metadata.AliasMetaData;
 import org.elasticsearch.common.collect.ImmutableOpenMap;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 import static org.elasticsearch.action.admin.indices.alias.IndicesAliasesRequest.AliasActions.*;
 
 @Repository
 public class IndexRepository {
 
-  private final TransportClient client;
+  private final RestHighLevelClient client;
 
   @Autowired
-  public IndexRepository(TransportClient client) {
+  public IndexRepository(RestHighLevelClient client) {
     this.client = client;
   }
 
   @SneakyThrows
   public String[] getIndices() {
-    return client.admin().indices()
-      .getIndex(new GetIndexRequest()).get()
+    return client.indices()
+      .get(new GetIndexRequest("*"), RequestOptions.DEFAULT)
       .getIndices();
   }
 
   @SneakyThrows
   public ImmutableOpenMap<String, List<AliasMetaData>> getAliasState() {
-    return client.admin().indices()
-      .getAliases(new GetAliasesRequest()).get()
-      .getAliases();
+    val aliases = client.indices().getAlias(new GetAliasesRequest(), RequestOptions.DEFAULT).getAliases();
+    val builder = new ImmutableOpenMap.Builder();
+
+    aliases.forEach((k,v) -> builder.put(k, v.stream().collect(Collectors.toList())));
```

Comment:
change to
`     aliases.forEach((k,v) -> builder.put(k, v.stream().collect(toUnmodifiableList()))); `